app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,9 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
```

Comment:
just needed simple user functionality to test in local host

Reply:
Yep - I ended up doing something similar on my branch. We'll have a bit of a merge conflict when I open mine. Merging this for now & we can figure out difference later.